src/api/mas/GetMAS.ts

```diff
@@ -0,0 +1,35 @@
+/* Import Dependencies */
+import axios from 'axios';
+
+/* Import Types */
+import { Dict, JSONResult } from 'app/Types';
+
+
+const GetMAS = async (handle: string) => {
+    let machineAnnotationService = {} as Dict;
+
+    if (handle) {
```

Comment:
what is this check for? if it's false, it looks like you still return `machineAnnotationService`, should you log an error here?

Reply:
It checks if a handle is provided, otherwise it won't make the call.
Yes I have to implement error handling here, that is exactly one of the improvements I am making in the refactoring process: error handling for fetch requests and returning exceptions :scream: 

I don't want to put too much effort into this now as I already changed the way of fetching in the first refactored code.
Will update this when I get to it in the refactoring process